rl_algo_impls/microrts/vec_env/microrts_vec_env.py

```diff
@@ -54,6 +54,7 @@ def __init__(
         cycle_maps=[],
         bot_envs_alternate_player: bool = False,
         video_frames_per_second: Optional[int] = None,
+        render_mode: Optional[str] = None,
```

Comment:
Same as above, define with render_mode already defaulted to rgb_array (render_mode: str = "rgb_array")